install/install_utils.php

```diff
@@ -982,23 +984,33 @@ function handleHtaccess()
     $restrict_str .= <<<EOQ
 RedirectMatch 403 {$ignoreCase}.*\.log$
 RedirectMatch 403 {$ignoreCase}/+not_imported_.*\.txt
-RedirectMatch 403 {$ignoreCase}/+(soap|cache|xtemplate|data|examples|include|log4php|metadata|modules)/+.*\.(php|tpl)
+RedirectMatch 403 {$ignoreCase}/+(soap|cache|xtemplate|data|examples|include|log4php|metadata|modules|vendor)/+.*\.(php|tpl)
 RedirectMatch 403 {$ignoreCase}/+emailmandelivery\.php
+RedirectMatch 403 {$ignoreCase}/+.git
+RedirectMatch 403 {$ignoreCase}/+.{$cacheDir}
+RedirectMatch 403 {$ignoreCase}/+tests
+RedirectMatch 403 {$ignoreCase}/+RoboFile\.php
+RedirectMatch 403 {$ignoreCase}/+composer\.json
+RedirectMatch 403 {$ignoreCase}/+composer\.lock
 RedirectMatch 403 {$ignoreCase}/+upload
 RedirectMatch 403 {$ignoreCase}/+custom/+blowfish
 RedirectMatch 403 {$ignoreCase}/+cache/+diagnostic
-RedirectMatch 403 {$ignoreCase}/+files\.md5$
-# END SUGARCRM RESTRICTIONS
+RedirectMatch 403 {$ignoreCase}/+files\.md5\$
+
 EOQ;
 
     $cache_headers = <<<EOQ
 
 <IfModule mod_rewrite.c>
     Options +SymLinksIfOwnerMatch
+    Options -Indexes
+    Options -MultiViews
     RewriteEngine On
     RewriteBase {$basePath}
     RewriteRule ^cache/jsLanguage/(.._..).js$ index.php?entryPoint=jslang&modulename=app_strings&lang=$1 [L,QSA]
     RewriteRule ^cache/jsLanguage/(\w*)/(.._..).js$ index.php?entryPoint=jslang&modulename=$1&lang=$2 [L,QSA]
+    RewriteRule ^cache/jsLanguage/(.._..).js$ index.php?entryPoint=jslang&module=app_strings&lang=$1 [L,QSA]
```

Comment:
where is module used, looking at the entry point it uses modulename, so I can see a reason for having both?

Reply:
These two lines were pulled from ```modules/Administration/UpgradeAccess.php```. I also don't see any obvious reason for this but I kept it in just in case there was some legacy backwards compatibility reason this was done.

Reply:
ok, but makes no sense it can't do both, I assume it just defaults to the first one so the second is irrelevant anyway

Reply:
Agreed, will remove.